### ISSUE TITLE
(Feature request) Use manually created quotes only

### ISSUE THREAD

**Issue by herrbrixter** [Lijukay/Qwotable#43]
I'd like to use only my own quotes. I already have a list of quotes that I will enter in the app and I want to cycle through in the widget.

Is this already possible in the settings? I just downloaded the app.

**Comment by herrbrixter** [comment]
FYI. I disabled "Include local Qwotables" and enabled "Include own Qwotables" but I see quotes other than my own.

**Comment by Lijucay** [comment]
I assume this is for the widget. I should clarify within the settings, that the selection is for the random quotes card on the app's home screen only.
But I'll totally add options for selecting the sources of quotes on the system's home screen widget.

**Comment by herrbrixter** [comment]
Yes, I'm referring to the widget. I thought the settings applied to both app homescreen and widget. Thanks!
